### ISSUE TITLE
How to create several taks with predefined activity for user story

### ISSUE THREAD

**Issue by MarkGTI** [figueiredorui/1-click-child-links#130]
Hi, 

We use a modified agile process

When I create a new user story I want to create 4 tasks

Requirements - Activity Requirements
Dev - Activity Dev
Testing - Activity Testing
Documentation - Activity Documentation

Figured out above bit, but I want to the title to be 

Dev - [User Story Title]
Tests - [User Story Title]
etc.

I'd also like to inherit other fields (due date, etc.)

is this possible?

Thanks
MArk

**Comment by mpburtt** [comment]
Yes. In the template, set the value for the title field to be "Dev - {System.Title}" or "Tests - {System.Title}", etc.
